packages/MarkdownEditor-Core.package/MarkdownToken.class/class/newFor.at..st

```diff
@@ -0,0 +1,7 @@
+instance creation
+newFor: aString at: startPosition
+
+	^ self new 
+		content: aString;
+		startPosition: startPosition;
+		yourself	
```

Comment:
tab after 'yourself'

Reply:
That's strange - I can see it in Git, but not in Squeak... I'll remove it either way